__tests__/Reviews.test.jsx

```diff
@@ -1,9 +1,19 @@
 // renders review title, text, ratings, reviewer name, date review is posted
 // should be have button to mark helpful
-import App from '../client/src/components/App';
+import Reviews from '../client/src/components/Reviews';
 
-describe('App', () => {
+const wrapper = shallow(<Reviews />);
+
+describe('<Reviews /> rendering', () => {
```

Comment:
Ditto to above comment, testing h3 is in review component vs Review being rendered in App.  Do you mean to test the component being rendered in App.  Maybe the test should be

const wrapper = shallow(`<App />`)
expect(wrapper.find(Reviews).toHaveLength(1)

Reply:
yes, these tests aren't implemented yet. This is just a skeleton for further development.

Reply:
awesome, thanks for the clarification.  Everything looks good =)